pkg/kinflate/util/util.go

```diff
@@ -111,6 +112,53 @@ func Encode(in map[GroupVersionKindName]*unstructured.Unstructured) ([]byte, err
 	return buf.Bytes(), nil
 }
 
+// SelectByGVK returns true if selector is a superset of in; otherwise, false.
```

Comment:
Please define superset with some examples.

its not obvious to me that if selector is nil, we have a superset condition.  is nil like a wildcard then?